packages/components/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const isDevelopment = process.env["NODE_ENV"] === "development";
+export const isProduction = process.env["NODE_ENV"] === "production";
```

Comment:
What's wrong with `process.env.NODE_ENV`?

Reply:
lint rule requiring indexes to be accessed as indexes

Reply:
Which lint rule? I don't think I saw an ESLint error

Reply:
some typescript strict flag